### PR TITLE
chore(solution): format core

### PR DESCRIPTION
**Pull request by 1yefuwang1**
format core

### CODE REVIEW

**Review by LongOddCode — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Review by IvanJobs — APPROVED** [review]

File: packages/fx-core/src/plugins/resource/localdebug/certificate.ts [hunk 1]
```diff
@@ -14,284 +22,307 @@ import * as ps from "./util/process";
 const continueText: string = "Continue";
 const learnMoreText: string = "Learn More";
 const learnMoreUrl: string = "https://aka.ms/teamsfx-ca-certificate";
-const confirmMessage: string = "To debug applications in Teams, your localhost server must be on HTTPS.\
+const confirmMessage: string =
+  'To debug applications in Teams, your localhost server must be on HTTPS.\
  For Teams to trust the self-signed SSL certificate used by the toolkit, a CA certificate must be added to your certificate store.\
- You may skip this step, but you'll have to manually trust the secure connection in a new browser window when debugging your apps in Teams.\
- For more information \"https://aka.ms/teamsfx-ca-certificate\". You may be asked for your account credentials when installing the certificate.";
+ You may skip this step, but you\'ll have to manually trust the secure connection in a new browser window when debugging your apps in Teams.\
```
Comment (swatDong) [inline packages/fx-core/src/plugins/resource/localdebug/certificate.ts:28]: Why this "\" added?

Comment (swatDong) [reply packages/fx-core/src/plugins/resource/localdebug/certificate.ts:28]: And I remember we should use `"` for string, not `'`

**Review by swatDong — COMMENTED** [review]
One comment for local debug part.
And... do merge and resolve conflict carefully...

**Review by XiaofuHuang — APPROVED** [review]

**Review by YitongFeng-git — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

File: packages/fx-core/src/plugins/resource/localdebug/certificate.ts (hunk 1 above)
Comment (1yefuwang1) [reply packages/fx-core/src/plugins/resource/localdebug/certificate.ts:28]: It's auto generated by prettier...

Comment (swatDong) [reply packages/fx-core/src/plugins/resource/localdebug/certificate.ts:28]: So what's the expected result, `"` or `'`?

**Review by a1exwang — APPROVED** [review]

File: packages/fx-core/src/plugins/resource/localdebug/certificate.ts (hunk 1 above)
Comment (1yefuwang1) [reply packages/fx-core/src/plugins/resource/localdebug/certificate.ts:28]: I think we are using ".

**Comment by zhenyasav** [comment]
Perhaps it might be easier to just run the command on `main` and push straight to main without a PR? this way we don't have to worry about conflicts?

File: packages/fx-core/src/plugins/resource/localdebug/certificate.ts (hunk 1 above)
Comment (swatDong) [reply packages/fx-core/src/plugins/resource/localdebug/certificate.ts:28]: Agree... then how to make sure next time it won't be auto generated to `'`...

**Comment by 1yefuwang1** [comment]
> Perhaps it might be easier to just run the command on `main` and push straight to main without a PR? this way we don't have to worry about conflicts? Theoretically that would reduce this PR down to any "real" changes we had in this (and other) PRs ?

Agreed. But I think only admins can do that. Currently managers and @zhenyasav are the admins.

**Pull request closed without merging**